.github/scripts/in_develop_labeler.py

```diff
@@ -4,8 +4,12 @@
 def main():
 	args = arg_getters.get_in_develop_labeler_args()
 	try:
+		#get pr body
+		pr_body = api_handler.get_pr_by_number(args.token, args.pr_num)["body"]
+
 		# find the issue closing line
-		issue_line = [line for line in args.body.split("\n") if line.startswith("**This PR closes")][0]
+		print(pr_body.split("\n"))
+		issue_line = [line for line in pr_body.split("\n") if line.startswith("**This PR closes")][0]
```

Comment:
Users could use multiple close statements. The complete list can be seen [**here**](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
To make the process easier, I thought of making an array including all the keywords, then checking if the PR contains one of those keywords.

Reply:
Yeah, agreed... though that is for another PR. I'll make an issue for it :D

Reply:
> Yeah, agreed... though that is for another PR. I'll make an issue for it :D

I'm not experienced in Python, but I could somehow try doing something if you'd like.

Reply:
I think we can improve it past just being keyword based. I made an issue here where we can discuss it further :)
https://github.com/devicons/devicon/issues/1578

I'll resolve this comment now, since I think it should be improved in a separate PR.